packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -6,6 +8,44 @@ const ADVANCED_GROUP = 'advanced';
 const DISPLAY_GROUP = 'stats';
 const ZERO_CONFIG_GROUP = 'zero-config';
 
+// Extract all the flages being exported from core. A list of cli flags generated by core
+// can be found here https://github.com/webpack/webpack/blob/master/test/__snapshots__/Cli.test.js.snap
+let flagsFromCore =
+    typeof cli !== 'undefined'
+        ? Object.entries(cli.getArguments()).map(([flag, meta]) => {
+              if (meta.simpleType === 'string') {
+                  meta.type = String;
+                  meta.usage = `--${flag} <value>`;
+              } else if (meta.simpleType === 'number') {
+                  meta.type = Number;
+                  meta.usage = `--${flag} <value>`;
+              } else {
+                  meta.type = Boolean;
+                  meta.negative = true;
+                  meta.usage = `--${flag}`;
+              }
+              return {
+                  ...meta,
+                  name: flag,
+                  group: 'core',
+              };
+          })
+        : [];
+
+// duplicate flags
+const duplicateFlags = ['entry', 'mode', 'stats', 'watch', 'target'];
+// broken flags
+const brokenFlags = ['module-no-parse', 'module-no-parse-reset', 'optimization-no-emit-on-errors', 'watch-options-poll'];
```

Comment:
`watch-options-poll` fixed

Reply:
`optimization-no-emit-on-errors` - what is bug with this?

Reply:
`webpack --optimization-no-emit-errors` should set `optimization.noEmitErrors: true` but it's not doing so.
![Screenshot at 2020-07-09 18-13-15](https://user-images.githubusercontent.com/46647141/87041672-37397880-c210-11ea-8ca5-81461f88545e.png)

Reply:
@snitin315 Let's send a fix to webpack, this option is very popular

Reply:
@evilebottnawi I investigated and found `optimization-no-emit-on-errors` comes with pre-appended value = true while importing and when we do `webpack --optimization-no-emit-on-errors` it sets it to flase.

Reply:
opposite behavior from rest of the boolean flags.

Reply:
Why we set it to false?

Reply:
we don't do this intentionally, I think somewhere we have bug I'm looking into it.

Reply:
I think that's probably caused by the `no-` instead of the name. Sounds like a bug in the args parsing.

Anyway it might make sense to change this in webpack core to `emitOnErrors`, as negative flags are bad practice anyway...

Reply:
@sokra yes, It was caused by `no-` it was a bug in `commander.js`, I have submitted a fix for same https://github.com/tj/commander.js/pull/1302 .

> Anyway it might make sense to change this in webpack core to emitOnErrors, as negative flags are bad practice anyway...

I Agree.

Reply:
This will be fixed with next release of `commander.js`.